File: Go/1732. Find the Highest Altitude.go
package main

//goland:noinspection GoUnusedFunction
func largestAltitude(gain []int) int {
	highest := 0
	current := 0

	for _, point := range gain {
		current += point
		if current > highest {
			highest = current
		}
	}

	return highest
}

File: Python/1732. Find the Highest Altitude.py
from typing import List


# noinspection PyMethodMayBeStatic
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        highest = 0
        current = 0

        for point in gain:
            current += point
            highest = max(highest, current)

        return highest
